winml/test/model/model_tests.cpp

```diff
@@ -173,8 +173,6 @@ std::string GetTestDataPath() {
 static std::vector<ITestCase*> GetAllTestCases() {
   std::vector<ITestCase*> tests;
   std::vector<std::basic_string<PATH_CHAR_TYPE>> whitelistedTestCases;
-  double perSampleTolerance = 1e-3;
```

Comment:
>doubl [](http://example.com/codeflow?start=2&length=5)

constexpr

Reply:
Sorry I'm not sure what you're asking me to do. Could you please elaborate?